src/components/TableModule/TableModule.stories.tsx

```diff
@@ -2,8 +2,14 @@ import React, { useRef } from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 
 import { TableModule } from './TableModule';
-import { TableConfiguration, TableSortClickProps } from './types';
+import {
+  RowSelectionState,
+  TableConfiguration,
+  TableSortClickProps,
+} from './types';
+import { ButtonUnstyled } from '../ButtonUnstyled';
```

Comment:
I removed it since row selection works in the storybook w/o it. Since the selection box is totally userspace thing, I'd like to keep it simple by using a `<Checkbox/>` only